ChangeLog.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [v0.5.1] - 2023-06-23
```

Comment:
@markus-liang-lrn @bhavya-shukla-lrn @markkellyeire I think we can probably disable this Codacy rule ("Lists should be surrounded by blank lines"), since it goes against our own convention in this file?

Reply:
Hi @TheRealEdDawson , which convention did you refer to? I see that the rule should be on based on this doc [https://learnosity.atlassian.net/wiki/spaces/DEV/pages/2171895868/Markdown+Linting+Rules+for+README+Markdown+Files+with+Codacy](https://learnosity.atlassian.net/wiki/spaces/DEV/pages/2171895868/Markdown+Linting+Rules+for+README+Markdown+Files+with+Codacy)